src/chatgpt-api.ts

```diff
@@ -307,7 +307,7 @@ Current date: ${currentDate}\n\n`
 
     const maxNumTokens = this._maxModelTokens - this._maxResponseTokens
     let { parentMessageId } = opts
-    let nextPromptBody = `${this._userLabel}:\n\n${message}<|im_end|>`
+    let nextPromptBody = `${this._userLabel}:\n\n${message}${this._completionParams.stop}`
```

Comment:
`stop` is an array of tokens, so this won't have the expected output right now

Reply:
You are completely right, sorry for that, I launched tests hoping they would catch that, should have watched more carefully :D